### PR TITLE
Build: Missing M55 include path

### PR DESCRIPTION
**Pull request by TTornblom**
Fixes issue causing compilation errors on IAR

Signed-off-by: TTornblom <thomas.tornblom@iar.com>

### CODE REVIEW

**Comment by grasci-arm** [comment]
Can one of the admins verify this patch?

File: ARM.CMSIS.pdsc [hunk 1]
```diff
@@ -2490,6 +2490,7 @@ and 8-bit Java bytecodes in Jazelle state.
       <files>
         <!-- include folder / device header file -->
         <file category="include"      name="Device/ARM/ARMCM55/Include/"/>
+        <file category="include"      name="Device/ARM/ARMCM55/Include/Template"/>
```
Comment (JonatanAntoni) [inline ARM.CMSIS.pdsc:2493]: This should not be required and may even break projects.
The `partition_ARMCM55.h` file is declared as a config file below. Hence is needs to be copied into the user project to be modifiably.

**Review by JonatanAntoni — CHANGES_REQUESTED** [review]
This change does not make sense. What does not work in IAR with the current component?

File: ARM.CMSIS.pdsc (hunk 1 above)
Comment (TTornblom) [reply ARM.CMSIS.pdsc:2493]: Sorry Jonatan, it was a secure/non-secure mismatch in my project.
I'll close the PR.

**Pull request closed without merging**